lms/envs/test.py

```diff
@@ -381,6 +381,22 @@
     'django.core.files.uploadhandler.TemporaryFileUploadHandler',
 )
 
+###################### Grade Downloads ######################
+# When tests run concurrently, make sure that they do not
+# interfere with each other when creating or removing temp files.
+GRADES_DOWNLOAD = {
+    'STORAGE_TYPE': 'localfs',
+    'BUCKET': 'edx-grades',
+    'ROOT_PATH': '/tmp/edx-s3-{}/grades'.format(THIS_UUID),
```

Comment:
I've been trying to use the PID, because it makes it slightly easier to debug. Also, I'm not sure if `settings.py` gets evaluated separately for each subprocess that nose forks off. I've been changing the root path in a `patch` call on each test, so that it happens during test startup... Although, now that I think about it, maybe the problem is that the `patch` call is getting executed when the tests are parsed, and then not getting re-evaluated when the test is running in a different process... Maybe it's worth a debugging assertion to validate that the current pid is in the filepath during test execution.